LOGGING_SYSTEM.md

```
@@ -180,11 +180,41 @@ Standardized error codes help identify common issues:
 - `EXTENSION_SYMLINK_FAILED`: Extension symlinking failed
 - `OWNERSHIP_CHANGE_FAILED`: File ownership change failed
 
+### Klipper Migration Error Codes:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)